cpp/cmake/thirdparty/get_faiss.cmake

```diff
@@ -115,4 +116,4 @@ endfunction()
 find_and_configure_faiss(
   BUILD_STATIC_LIBS ${RAFT_USE_FAISS_STATIC}
   ENABLE_GPU ${RAFT_FAISS_ENABLE_GPU}
-)
+)
```

Comment:
I noticed several files in this PR that do not end with a newline.

In practice I've found that this rarely matters, but it can occasionally cause issues for tools doing line-by-line parsing of files. In a future PR, you may want to consider automatically fixing these with `pre-commit`, like `cudf` does: https://github.com/rapidsai/cudf/blob/107753ccaacdb62287c4dd4351e5caf3bf8bc62a/.pre-commit-config.yaml#L13